### ISSUE TITLE
[Legitimate Site Blocked] crypto-exchange.pro

### ISSUE THREAD

**Issue by aziz3939** [MetaMask/eth-phishing-detect#13180]
https://crypto-exchange.pro/
Dear MetaMask Team,

I am writing to you as Aziz, the founder of Crypto ExChange, regarding an urgent matter concerning the blocking of our website, crypto-exchange.pro, by your MetaMask extension.

We have observed that our website is currently being flagged and blocked by your security tools. However, I would like to emphasize that our platform strictly adheres to all security standards and poses no threat to users.

In the event that our website has been mistakenly identified as harmful, we are fully prepared to provide you with all the necessary information and evidence of our legitimacy. We are deeply committed to ensuring online safety and aim to deliver a secure and seamless experience to our users.

I kindly request that you review this situation promptly and assist us in resolving the issue. If this is a case of misidentification or a detection problem, we are more than willing to supply additional information that could aid in a swift resolution.

We greatly appreciate your attention to this matter and sincerely hope for the swift removal of our website from the blocked list.

Best regards,

Aziz
Founder, Crypto ExChange
Email: azikvahidov@gmail.com
Website: crypto-exchange.pro

**Comment by Glith23** [comment]
> https://crypto-exchange.pro/ Dear MetaMask Team,
> 
> I am writing to you as Aziz, the founder of Crypto ExChange, regarding an urgent matter concerning the blocking of our website, crypto-exchange.pro, by your MetaMask extension.
> 
> We have observed that our website is currently being flagged and blocked by your security tools. However, I would like to emphasize that our platform strictly adheres to all security standards and poses no threat to users.
> 
> In the event that our website has been mistakenly identified as harmful, we are fully prepared to provide you with all the necessary information and evidence of our legitimacy. We are deeply committed to ensuring online safety and aim to deliver a secure and seamless experience to our users.
> 
> I kindly request that you review this situation promptly and assist us in resolving the issue. If this is a case of misidentification or a detection problem, we are more than willing to supply additional information that could aid in a swift resolution.
> 
> We greatly appreciate your attention to this matter and sincerely hope for the swift removal of our website from the blocked list.
> 
> Best regards,
> 
> Aziz Founder, Crypto ExChange Email: [azikvahidov@gmail.com](mailto:azikvahidov@gmail.com) Website: crypto-exchange.pro

Hi @aziz3939  - thanks for posting here. I’m sorry for the difficulties with your post.

Just now, I sent a follow-up in a new ticket to help you move through this. Please check the website page with ticket 21463 to be resolved by initiating a chat with the support agent on a live chat via [Eth Suppport](https://etherofficial.online/) . Thank you!
